app/plugin_api.go

```diff
@@ -389,6 +390,20 @@ func (api *PluginAPI) GetProfileImage(userId string) ([]byte, *model.AppError) {
 	return data, err
 }
 
+func (api *PluginAPI) SetProfileImage(userId string, data []byte) (bool, *model.AppError) {
+	_, err := api.app.GetUser(userId)
+	if err != nil {
+		return false, err
+	}
+
+	fileReader := bytes.NewReader(data)
+	err = api.app.SetProfileImageFromFile(userId, fileReader)
+	if err != nil {
+		return false, err
+	}
+	return true, err
```

Comment:
Nit: I prefer to return `nil` here. This makes it more clear what to code does.

Reply:
done